packages/server/migrations/js/201912230755000-notebooks-index.js

```diff
@@ -32,6 +35,10 @@ module.exports = {
         'Notebooks_name_ownerId_unique',
       );
 
+      // @todo: Here first of all we should have to remove
+      // @todo: duplicates and only then to create another
+      // @todo: unique index.
```

Comment:
When do you expect that we do these @todos? Can you make a new story or two and prioritize accordingly?

Reply:
It is a todo not in a direct sense. It is like a remark. I am not sure that we should do something here this is why I did not create a new issue.

Reply:
Ok, that works